README.md

```diff
@@ -19,42 +19,27 @@ Kong runs natively on Kubernetes thanks to its official [Kubernetes Ingress Cont
+We suggest using the getting started script located in our [docs](https://docs.konghq.com/gateway/latest/get-started/). 
 
-Whether you’re running in the cloud, on bare metal, or using containers, you can find every supported distribution on our [official installation](https://konghq.com/install/#kong-community) page.
+Whether you’re running in the cloud, on bare metal, or using containers, you can find every supported distribution on our [installation options] (https://docs.konghq.com/gateway/latest/install/) page.
 
-1) To start, clone the Docker repository and navigate to the compose folder.
-```cmd
-  $ git clone https://github.com/Kong/docker-kong
-  $ cd docker-kong/compose/
-```
-
-2) Start the Gateway stack using:
-```cmd
-  $ KONG_DATABASE=postgres docker-compose --profile database up
-```
-
-The Gateway will be available on the following ports on localhost:
 
-`:8000` on which Kong listens for incoming HTTP traffic from your clients, and forwards it to your upstream services.
-`:8001` on which the Admin API used to configure Kong listens.
+## Features
 
-Next, follow the [quick start guide](https://docs.konghq.com/gateway-oss/latest/getting-started/configuring-a-service/
-) to tour the Gateway features.
+Kong Gateway is a lightweight, fast, and flexible cloud-native API gateway. An API gateway is a reverse proxy that lets you manage, configure, and route requests to your APIs.
 
-## Features
+Kong Gateway runs in front of any RESTful API and can be extended through modules and plugins. It’s designed to run on decentralized architectures, including hybrid-cloud and multi-cloud deployments.
 
-By centralizing common API functionality across all your organization's services, the Kong API Gateway creates more freedom for engineering teams to focus on the challenges that matter most. 
+With Kong Gateway, users can:
 
-The top Kong features include:
-- Advanced routing, load balancing, health checking - all configurable via a RESTful admin API or declarative configuration.
-- Authentication and authorization for APIs using methods like JWT, basic auth, OAuth, ACLs and more.
-- Proxy, SSL/TLS termination, and connectivity support for L4 or L7 traffic.
-- Plugins for enforcing traffic controls, rate limiting, req/res transformations, logging, monitoring and including a plugin developer hub.
-- Sophisticated deployment models like Declarative Databaseless Deployment and Hybrid Deployment (control plane/data plane separation) without any vendor lock-in.
+- Leverage workflow automation and modern GitOps practices
+- Decentralize applications/services and transition to microservices
+- Create a thriving API developer ecosystem
+- Proactively identify API-related anomalies and threats
+- Secure and govern APIs/services, and improve API visibility across the entire organization.
```

Comment:
This is a bit too fluffy - there are words that don't technically mean much.
I recommend keeping this technical as much as possible because the audience that looks at this level is generally very technical in nature.